biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -476,6 +477,14 @@
 	 */
 	String								TSTAMP										= "_@tstamp";
 
+	/*
+	 * Module Section
+	 */
+
+	String								MODULE_INFO_CLASS							= "module-info.class";
+	String								MODULE_INFO_JAVA							= "module-info.java";
```

Comment:
These two constants (_JAVA and _NAME) are unused by this PR. Perhaps it is better to define them later when actually needed?

Reply:
will remove